docs/UserGuide.adoc

```diff
@@ -231,23 +206,24 @@ Examples:
 === News Feed
 ==== Adding a feed : `addfeed`
 
-Adds a new feed to the news feed. Expects the URL of an XML feed. +
-Format: `addfeed \n [name of feed] \u [url]`
+Adds a feed to the news feed. +
+Format: `addfeed \n [name of feed] \a [address of the RSS feed]`
 
 Examples:
 
-* `addfeed \n [Seth Lui] \u https://sethlui.com/feed`
+* `addfeed \n [Eatbook] \a https://eatbook.com/feed`
 
-==== Removing a feed : `removefeed`
+==== Deleting a feed : `deletefeed`
 
-Removes a feed from the news feed. +
-Format: `removefeed \n [name of feed]`
+Deletes a feed from the news feed. +
+Format: `deletefeed \n [name of feed]`
 
 Examples:
 
-* `removefeed \n Seth Lui`
+* `deletefeed \n Eatbook`
 
```

Comment:
Mode command - Toggle between Main mode and Todo mode.
Format: mode
Add command - Add a new eatery to todo list. Do not require the input of category.
Format: add \n Two Chefs Eating Place \a 116 Commonwealth Cres, #01-129 \t cheap